dvc/path/local.py

```diff
@@ -8,4 +8,7 @@ class PathLOCAL(PathBASE):
     scheme = Schemes.LOCAL
 
     def __str__(self):
-        return os.path.relpath(self.path)
+        try:
+            return os.path.relpath(self.path)
+        except ValueError:
```

Comment:
Let's also put a comment about it. It is caused by os.path.relpath not being able to find a relpath between drives on windows, right?